lib/matplotlib/stackplot.py

```diff
@@ -23,14 +23,14 @@ def stackplot(axes, x, *args,
 
     Parameters
     ----------
-    x : (N,) array-like
+    x : (N, ) array-like
```

Comment:
```suggestion
    x : (N,) array-like
```
No space here. Compare 
```
In [2]: np.zeros(5).shape
Out[2]: (5,)
```